documentation/staking-governance.md

```diff
@@ -13,37 +13,53 @@ This design with selectable locking duration helps to align token holders with t
   - Confiscated tokens are immediately redirected to the staking rewards pool to be distributed over the next 14 days period on top of the guaranteed rewards.
   - Users do not have the option to cancel the fast un-stake during this 3 day window.
 
 ## Voting Power
 
-The voting power for a locked amount of tokens is computed as follows:
+The voting power of an account is expressed by the account's balance of another ERC20 contract, the *Voting Power* contract. The token symbol of this contract `vpLSK`. The voting power for a locked amount of tokens is computed as follows:
 
-- Generally, locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides one unit of voting power.
-- In case a user pauses their locking period countdown, they receives a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200%.
+- Generally, locked tokens provide a voting power proportional to the amount of locked tokens. Concretely, 1 locked LSK provides 1 vpLSK.
+- In case a user pauses their locking period countdown, they receive a boost of the voting power meaning the voting power is set to `lockedAmount * (1 + remainingLockingDurationInDays/365)`. Hence, the voting power can be increased by up to 200% by pausing a locking position with the maximum locking period of 2 years.
+
+### Voting Power for a Concrete Proposal
+
+At the time a proposal is created, a snapshot of the delegated voting power is taken and the voting result is based on this snapshot. Consequently, neither any changes in the voting power nor any delegations made after the proposal creation have any influence on the voting result.
 
 ## Staking Rewards
 
 Users receive rewards for their locking tokens as follows: 
 
 - Rewards are calculated on a daily basis, based on the amount locked and the remaining locking duration. Concretely, the weight for a locked amount is given by `lockedAmount * (remainingLockingDurationInDays + 150)`. The total daily amount of staking rewards is then shared by all users proportional to their weight.
-- Users can claim their rewards (and restake them immediately if they want) at any time.   
+- Users can claim their rewards (and restake them immediately if they want) at any time.
+
+## Pausing a Locking Position
+
+As mentioned in several subsection before, it is possible *to pause* the countdown of a locking position. This can be done at any time. Once paused, the remaining locking duration does not decrease anymore. The owner of the locking position can *resume* the countdown of the locking period an any point in time. Once resumed, the remaining locking duration descreases again daily.
```

Comment:
"subsection" --> "subsections". We could also say "several places above", but again no strong opinion.

Reply:
done